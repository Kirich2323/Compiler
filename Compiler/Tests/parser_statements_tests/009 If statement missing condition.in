var
    a : integer;
begin
    if then
    begin
        a := 10;
    end
end.