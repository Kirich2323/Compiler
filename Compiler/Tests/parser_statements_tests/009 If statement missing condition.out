(4 ; 8): Invalid expression.